packages/camera/camera_avfoundation/ios/Classes/FLTCam.h

```diff
@@ -14,6 +14,35 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * A class that manages media settings state and performs media settings operations.
+ */
+@interface FLTCamMediaSettingsProvider : NSObject
+
+@property(atomic, readonly, strong) NSNumber *fps;
```

Comment:
`FPS`

Reply:
Abbreviated `fps` is changed to `framesPerSecond` in `camera-avfoundation` implementation. Ok?